### PR TITLE
Add unit test TestListPlugins for the scheduler

### PR DESCRIPTION
**Pull request by AIchovy**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR adds UT for scheduler framework
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: pkg/scheduler/framework/v1alpha1/framework_test.go [hunk 1]
```diff
@@ -1701,6 +1701,48 @@ func TestWaitOnPermit(t *testing.T) {
 	}
 }
 
+func TestListPlugins(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins *config.Plugins
+		// If noPlugin is true, we expect 'ListPlugins' method return empty plugin except queue sort plugin and bind plugin.
+		noPlugin bool
```
Comment (draveness) [inline pkg/scheduler/framework/v1alpha1/framework_test.go]: what about pluginsCount

**Review by draveness — COMMENTED** [review]
The code is not formatted, let's see what the CI would tell us.

/ok-to-test

File: pkg/scheduler/framework/v1alpha1/framework_test.go (hunk 1 above)
Comment (mosesyou) [reply pkg/scheduler/framework/v1alpha1/framework_test.go]: Do you mean adding the 'pluginsCount'  as a test parameter?

Comment (mosesyou) [reply pkg/scheduler/framework/v1alpha1/framework_test.go]: I have added `pluginsCount` and remove `noPlugin`.

**Comment by mosesyou** [comment]
replaced by #88717

**Pull request closed without merging**